crm_integration_xblock/crm_integration_xblock.py

```diff
@@ -32,6 +38,10 @@ class CrmIntegration(StudioEditableXBlockMixin, XBlock):
         default="Crm Integration"
     )
 
+    name = String(help="Please write the name of your company",
```

Comment:
esta variable sería mucho más descriptiva si se llamara backend_name. Y en help puedes poner "Please write the name of the backend. Normally it is the name of the project you are working on"

Reply:
precisamente para el caso en el que varkey pueda tener más de un backend